core/domain/exp_domain_test.py

```diff
@@ -10789,7 +10789,6 @@ def _get_default_state_dict(content_str, dest_name):
                 },
                 'param_changes': [],
             }
-
```

Comment:
This change is not related to this PR, right?

Reply:
No It must have been a mistake. Very sorry for the inconvinience!!

Reply:
> Thanks @Mayank-gaur! The tests look great now. You reverted one change, and have an unaddressed comment (please comment "Done" only when you have done the change). Once you fix those, I will approve the PR.

@sajalasati can you please tell me which unaddressed comment are you talking about? Is is about the test cases names? Because I did corrected it by looking at others name.

Reply:
I was talking about this comment only.